src/main/java/com/google/gfx/model/Location.java

```diff
@@ -0,0 +1,29 @@
+package com.google.gfx.model;
+
+/**
+ * Used to localise users.
+ */
+public enum Location {
```

Comment:
Perhaps make it a class rather than an enum. We may want to pull the list of locations from some service instead of hard-coding them.